debezium-core/src/main/java/io/debezium/relational/Column.java

```diff
@@ -155,6 +155,13 @@ default boolean isRequired() {
      */
     String comment();
 
+    /**
+     * Get whether the column maps to an invisible column.
+     *
+     * @return {@code true} if the column is invisible, or {@code false} otherwise
+     */
+    boolean isInvisible();
```

Comment:
I wonder if using the name `invisible` is correct. IMHO `INVISIBLE`  columns in Oracle, MySQL etc. is something completely different and we are processing them. Is not `ROWID` pseudo (synthetic) column, not invisible?